MentoriaiOS.playground/Contents.swift

```diff
@@ -10,55 +10,67 @@
  verificar se tem letra maiúscula nesta palavra
 */
 
-// MARK: - Properties
-
-print("Quero conflitos 😈")
-
-var passwordsToValidate = ["#forTe1", "senhafraca", "Qu@s1", "Voce@Consegue!2019"]
-var newPasswordsMaiscule = [String]()
-// o que é Maiscule?
-var newPasswordsMinuscule = [String]()
-
-// MARK: - Methods
-
-private func validatePassword(with password: [String]) -> [String] {
-    // aqui percorre a lista inteira
-    return passwordsToValidate.filter { $0.count >= 7 }
-}
-
-validatePassword(with: passwordsToValidate)
-
-// Valida se possui pelo menos uma letra minúscula.
-for password in passwordsToValidate {
-    // aqui percorre a lista inteira
-    for character in password {
-        if character.isLowercase {
-            newPasswordsMinuscule.append(password)
-            break
+class PasswordValidation {
+    
+    // MARK: - Public Methods
+    
+    func validate(passwords: [String]) -> [String] {
+        
+        var filtersValidePasswords = passwords
+        
+        filtersValidePasswords = filterMinimumSevenCharacters(with: filtersValidePasswords)
+        filtersValidePasswords = filterAtLeastOneUpperCaseCharacters(with: filtersValidePasswords)
+        filtersValidePasswords = filterAtLeastOneLowerCaseCharacters(with: filtersValidePasswords)
+        
+        return filtersValidePasswords
+    }
+    
+    // MARK: - Private Methods
+    
+    private func filterMinimumSevenCharacters(with passwords: [String]) -> [String] {
+        let passwords = passwords.filter { $0.count >= 7 }
```

Comment:
se for só essa linha mesmo, já pode dar o return aqui mesmo, nem precisaria do let!
vc está atribuindo nessa variável e nem tá usando